go.mod

```diff
@@ -177,6 +177,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchpersi
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/experimentalmetricmetadata => ./pkg/experimentalmetricmetadata
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator => ./pkg/translator
```

Comment:
No need for this

Reply:
Also incorrect should be `replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger
 => ./pkg/translator/jaeger` you can keep it here this way because you need it for jaeger components

Reply:
yeah sure, I will make the changes